src/components/carousel/_index.scss

```diff
@@ -0,0 +1,152 @@
+@use "../../scss";
+@use "~slick-carousel/slick/slick";
+
+.c-carousel {
+	@include scss.component-properties("carousel");
+
+	/* Arrows */
+	.slick-prev,
+	.slick-next {
+		cursor: pointer;
+		font-size: 0;
+		line-height: 0;
+		position: absolute;
+		top: 50%;
+		display: block;
+
+		&:hover {
+			color: transparent;
+			outline: none;
+			background: transparent;
+
+			&:before {
+				opacity: 1;
+			}
+		}
+
+		&:focus {
+			color: transparent;
+			outline: none;
+			background: transparent;
+
+			&:before {
+				opacity: 1;
+			}
+		}
+
+		@include scss.component-properties("carousel-buttons");
+	}
+
+	.slick-next {
+		@include scss.component-properties("carousel-button-next");
+
+		&:before {
+			font-family: "Arial", serif;
+			content: ">";
+			-webkit-font-smoothing: antialiased;
+			-moz-osx-font-smoothing: grayscale;
+
+			@include scss.component-properties("carousel-button-next-before");
+		}
+	}
+
+	.slick-prev {
+		@include scss.component-properties("carousel-button-previous");
+
+		&:before {
+			font-family: "Arial", serif;
+			content: "<";
+			-webkit-font-smoothing: antialiased;
+			-moz-osx-font-smoothing: grayscale;
+
+			@include scss.component-properties("carousel-button-previous-before");
```

Comment:
this can be added to the parent if you want as a single include line (after this block)
```
@include scss.component-with-states("carousel-button-previous", [before])
```